nifi-commons/nifi-property-protection-factory/src/main/java/org/apache/nifi/properties/scheme/PropertyProtectionScheme.java

```diff
@@ -19,7 +19,7 @@
 /**
  * Property Protection Schemes supported as arguments for encryption commands should not have direct references
  */
-enum PropertyProtectionScheme implements ProtectionScheme {
+public enum PropertyProtectionScheme implements ProtectionScheme {
```

Comment:
I added a SchemeCandidates class which utilises the StandardProtectionSchemeResolver instead of directly using the PropertyProtectionScheme enum. Does this resolve the issue at hand?